### PR TITLE
Switch back to actual django-modeltranslation

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/__init__.py [hunk 1]
```diff
@@ -3,3 +3,8 @@
 # This will make sure the app is always imported when
 # Django starts so that shared_task will use this app.
 from .temba_celery import app as celery_app  # noqa
+
+# Patch django-modeltranslation's get_language to not check the active language is configured in Django
+from modeltranslation import utils
+
+utils.get_language = lambda: utils._get_language()
```
Comment (rowanseymour) [inline temba/__init__.py]: As far as I can see this is the only place where our old fork diverges so am thinking it's better to just patch this https://github.com/nyaruka/django-modeltranslation/commit/dffebc9363297d77d91c99d46e8e53b32dc05e61

**Comment by nicpottier** [comment]
Pretty sure we don't use this anywhere.. is it being pulled in by Smartmin? Can we make it optional at that level perhaps?

**Comment by rowanseymour** [comment]
Doesn't appear to be a smartmin thing (https://github.com/nyaruka/smartmin/blob/master/requirements/base.txt). Guess we can pull it and see what happens

**Comment by rowanseymour** [comment]
Replaced by #1071

**Pull request closed without merging**